tests/topotests/bgp_srv6l3vpn_to_bgp_vrf3/test_bgp_srv6l3vpn_to_bgp_vrf3.py

```diff
@@ -95,7 +95,7 @@ def open_json_file(filename):
         assert False, "Could not read file {}".format(filename)
 
 
-def check_rib(name, cmd, expected_file):
+def check_rib(name, cmd, expected_file, count=10, wait=0.5):
```

Comment:
We expect 15 seconds as a minimum, can we align here also?

Reply:
fixed